AppSyncRealTimeClient/Support/AppSyncURLHelper.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2018-2021 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct AppSyncURLHelper {
+
+    public static let standardDomainPattern =
+    "^https://\\w{26}.appsync-api.\\w{2}(?:(?:-\\w{2,})+)-\\d.amazonaws.com/graphql$"
```

Comment:
nit: add comment for describing what an AppSync standard domain url look like, and example of this, can tie it back to what regex components